### PR TITLE
support resize, nonmaxsuppression in opset 10; and fix some bugs.

### PR DESCRIPTION
**Pull request by zhijxu-MS**

### CODE REVIEW

**Comment by nbcsm** [comment]
https://github.com/onnx/tensorflow-onnx/issues/468

File: tf2onnx/onnx_opset/nn.py [hunk 1]
```diff
@@ -464,14 +464,15 @@ def version_7(cls, ctx, node, **kwargs):
...
+        cls._convert_since_9(ctx, node, node_type="Upsample")
 
     @classmethod
     def version_10(cls, ctx, node, **kwargs):
-        cls._convert_since_9(ctx, node, **kwargs)
+        cls._convert_since_9(ctx, node, node_type="Resize")
 
     @classmethod
-    def _convert_since_9(cls, ctx, node, **kwargs):
+    def _convert_since_9(cls, ctx, node, node_type):
```
Comment (nbcsm) [inline tf2onnx/onnx_opset/nn.py]: def _convert_since_9(cls, ctx, node, onnx_op, **kwargs):

cls._convert_since_9(ctx, node, onnx_op="Upsample", **kwargs)

cls._convert_since_9(ctx, node, onnx_op="Resize", **kwargs)

File: tf2onnx/onnx_opset/tensor.py [hunk 2]
```diff
@@ -922,3 +922,30 @@ def version_4(cls, ctx, node, **kwargs):
...
+        ctx.insert_new_node_on_input(node, "Unsqueeze", node.input[1], axes=[0, 1])
+        ctx.insert_new_node_on_input(node, "Cast", node.input[2], to=onnx_pb.TensorProto.INT64)
+        # replace original node with nonmaxsurppress + slice + squeeze + cast
+        dtypes = [ctx.get_dtype(node.output[0])]
+        shapes = [ctx.get_shape(node.output[0])]
+        ctx.remove_node(node.name)
+        new_nonmaxsurppress = ctx.make_node(node.type, node.input).output[0]
+        attr = {"axes": [1], "ends": [3], "starts": [2]}
+        inputs_map = {"data": new_nonmaxsurppress, **attr}
+        slice_op = GraphBuilder(ctx).make_slice(inputs_map)
```
Comment (nbcsm) [inline tf2onnx/onnx_opset/tensor.py]: [optional] 1 line?
slice_op = GraphBuilder(ctx).make_slice({"data": new_nonmaxsurppress, "axes": [1], "ends": [3], "starts": [2]})

File: tf2onnx/onnx_opset/nn.py (hunk 1 above)
Comment (nbcsm) [inline tf2onnx/onnx_opset/nn.py]: node_type => op_type?

File: tf2onnx/onnx_opset/tensor.py (hunk 2 above)
Comment (zhijxu-MS) [reply tf2onnx/onnx_opset/tensor.py]: done

File: tf2onnx/tfonnx.py [hunk 3]
```diff
@@ -471,9 +471,7 @@ def rewrite_incomplete_type_support_rs5(g, ops):
 def rewrite_incomplete_type_support_rs6(g, ops):
     impacted_ops = [
         "Div",
-        "Greater",
```
[the remark this answers is not in the document]
Comment (zhijxu-MS) [reply tf2onnx/tfonnx.py]: Yuanhong has a [PR](https://github.com/onnx/tensorflow-onnx/pull/431/files), he refines the logic to support incomplete dtype in ops' conversion function.

**Pull request merged**